hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/SlopFixingCandidateGenerator.java

```diff
@@ -89,7 +97,7 @@ BalanceAction generateCandidate(BalancerClusterState cluster, boolean isWeighing
         fixedServers.forEach(s -> cs.getServersByLoad().remove(s));
         fixedServers.clear();
         if (!regionFoundMove) {
-          LOG.debug("Could not find a destination for region {} from server {}.", regionIdx,
+          LOG.trace("Could not find a destination for region {} from server {}.", regionIdx,
```

Comment:
Since its nested in double for-loops, probably better to wrap it in a `if(isTraceEnabled()` branch. I have no micro-benchmark to support this suggestion.